okapi-core/src/main/java/org/folio/okapi/bean/ModuleInstance.java

```diff
@@ -44,6 +44,10 @@ public ModuleInstance(ModuleDescriptor md, RoutingEntry re,
     this.withRetry = false;
   }
 
+  public void substPath(String pattern, String value) {
+    path = path.replace(pattern, value);
+  }
+
```

Comment:
Are there plans for other substitutions or can we use
```
/**
 * Substitute {id} in path
 */
public void substPathId(String id) {
  path = path.replace("{id}", id);
}
```

Reply:
Not now. Can change it, if you like.

Reply:
I like it. Otherwise the task is split: Call to replace in ModuleInstance, substitution string "{id}" in TenantManager.

Reply:
Done.